node/src/accountant/big_int_db_processor.rs

```diff
@@ -197,11 +233,12 @@ impl<'a, T: DAOTableIdentifier> BigIntSqlConfig<'a, T> {
             .skip_while(|char| char.is_whitespace())
             .take_while(|char| !char.is_whitespace())
             .collect::<String>();
-        match (keyword.trim(), self.update_clause_opt.is_some()) {
-            ("insert", true) => "upsert".to_string(),
-            ("update", false) => keyword,
+        match keyword.trim() {
```

Comment:
Given the `skip_while` and `take_while` above, you may be able to remove this `trim`.